src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -143,9 +138,20 @@
                 name: 'language'
               }
             } only %}
-            <span class="ecl-site-header__language-code">{{ _language.code }}</span>
+            <span class="ecl-site-header__language-code">{{ _language_selector.code }}</span>
           </span>
         </a>
+        {% include '../ec-component-language-list/language-list-overlay.html.twig' with { 
+          items: _language_selector.overlay.items,
```

Comment:
But this means that every implementation of this site header would need the developers to pass all the list of items in order to build the language list? I mean, this is a language switcher, it should ask you for the current url and for the rest it should work autonomously, i think, the language list is predefined as its order is, another parameter could be the type of language negotiation (suffix, prefix, underscore, slash, whatever...) and then it would build the list without needing to be fed, what do you think? :)

Reply:
As agreed we will consider this for a future release.. ;)